chore: Update documentation for installing EFA device plugin to use official EKS chart (#1904)
aws-ia/terraform-aws-eks-blueprints: patterns/elastic-fabric-adapter/README.md

## Validate

!!! note

>     The following steps are shown with `g5.8xlarge` due to cost and availability. Values shown below will change based on the instance type selected (i.e. - `p5.48xlarge` has 8 GPUs and 32 EFA interfaces)

Reviewer: ```suggestion
    The following steps are shown with `g5.8xlarge` for frugality. Values shown below will change based on the instance type selected (i.e. - `p5.48xlarge` has 8 GPUs and 32 EFA interfaces)
```
